dune-project

```diff
@@ -2,7 +2,7 @@
 
 (name current-web-pipelines)
 
-(generate_opam_files true)
+(generate_opam_files false)
```

Comment:
:open_mouth:

Reply:
When enabled it overwrites the `pin-depends` in the opam file :(
Is there another way of doing it?

Reply:
I think you can write a `current-web-pipelines.opam.template` file containing
```
pin-depends: [
...
]
```

it will be added to the generated opam file